sound/soc/sof/ipc.c

```diff
@@ -460,9 +460,6 @@ static void ipc_period_elapsed(struct snd_sof_dev *sdev, u32 msg_id)
 		posn_offset = spcm->posn_offset[direction];
 		snd_sof_dsp_mailbox_read(sdev, posn_offset, &posn,
 					 sizeof(posn));
-
-		dev_dbg(sdev->dev, "posn mailbox: posn offset is 0x%x",
```

Comment:
we still need this for debugging, maybe change it to verbose version?

Reply:
@keyonjie why do you need this information repeated if it's constant after hw_params?

Reply:
sorry, I thought it was posn, for the constant posn offset, we can remove it.